README.md

```diff
@@ -1,7 +1,5 @@
 # The Things Stack, an open source LoRaWAN Network Server
 
-[![Build Status](https://travis-ci.com/TheThingsNetwork/lorawan-stack.svg?branch=master)](https://travis-ci.com/TheThingsNetwork/lorawan-stack) [![Coverage Status](https://coveralls.io/repos/github/TheThingsNetwork/lorawan-stack/badge.svg?branch=master)](https://coveralls.io/github/TheThingsNetwork/lorawan-stack?branch=master)
```

Comment:
We can't (won't) update this to always point to the latest version, so instead of confusing people, I think it's better to remove these.